team_wizz.js

```diff
@@ -1 +1,33 @@
 // Fetch all routes for the airline and insert into #wizz
+
+(async () => {
+  const firstFlight = await fetch("https://flyapi.onrender.com/tlv_ber/wizz");
+  const secondFlight = await fetch("https://flyapi.onrender.com/arn_lhr/wizz");
+  const thirdFlight = await fetch("https://flyapi.onrender.com/lis_nyc/wizz");
+
+  const tlvBer = await firstFlight.json();
+  const arnLhr = await secondFlight.json();
+  const lisNyc = await thirdFlight.json();
+
+  const wizzContainer = document.getElementById("wizz");
```

Comment:
I would also separate the DOM related code into a separate function - so if you want to change it it will be done in one place.

Reply:
This review wasn't too clear, but I have tried to refactor the code to my own understanding